Nature Publishing Group.js

```diff
@@ -1419,26 +1419,22 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"date": "November 2006",
+				"date": "2006-11",
 				"DOI": "10.1038/ng1901",
-				"ISSN": "1061-4036",
+				"ISSN": "1546-1718",
 				"abstractNote": "The estrogen receptor is the master transcriptional regulator of breast cancer phenotype and the archetype of a molecular therapeutic target. We mapped all estrogen receptor and RNA polymerase II binding sites on a genome-wide scale, identifying the authentic cis binding sites and target genes, in breast cancer cells. Combining this unique resource with gene expression data demonstrates distinct temporal mechanisms of estrogen-mediated gene regulation, particularly in the case of estrogen-suppressed genes. Furthermore, this resource has allowed the identification of cis-regulatory sites in previously unexplored regions of the genome and the cooperating transcription factors underlying estrogen signaling in breast cancer.",
 				"issue": "11",
-				"journalAbbreviation": "Nat Genet",
+				"journalAbbreviation": "Nature Genetics",
```

Comment:
It looks like the journal abbreviations are all wrong now, let's just drop the variable from the translator.